.github/workflows/set-milestone-on-referenced-issue.yml

```diff
@@ -13,10 +13,6 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
please change name from `name: 'Milestone for PR'`
to name: 'Milestone issue closed by PR'